### PR TITLE
DOCS: PLGLTS2

### PR DESCRIPTION
**Pull request by Lars-MultiSafepay**
### Basic
- We only accept documentation that is proved to work on our LIVE API
- At least 1 approval is needed before pull requests can be merged
- The article must match our Synonym word list. For external commits, a MultiSafepay employee will check
- US English must be applied
- Use bullets for numerations where the sort order doesn't matter. For chronological order, use numbers. This does not apply to changelogs
- Set link on email address.

### Text style
- Paths are written like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in your Markdown code (example title: 'Title of the article')
- Use Italics for buttons, clicks, paths, etc. Do mind to close the italics before the end of a sentence followed by a period
- Numerations / bullets are punctuated with a period at the end of the numerations / bullets.
- Periods should be excluded for sentences that finish in an e-mail address (e.g. "..contact us at <example@example.com> "  )

### Images
- Screenshot recomended sizes: Width: 820px - Height: auto

### CODE REVIEW

File: content/integrations/hosted/lightspeed_app/faq/change-environment.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title : "Can I change the API key after installation?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+
+You sure can. In the settings page click on _Environment & API key_. You will be redirected to another page.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/change-environment.md:7]: Yes. On the settings page click on _Environment & API key_. You will be redirected to another page.

File: content/integrations/hosted/lightspeed_app/faq/change-environment.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title : "Can I change the API key after installation?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+
+You sure can. In the settings page click on _Environment & API key_. You will be redirected to another page. 
+Here you can update the API key or the environment, or both.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/change-environment.md:8]: Here you can update the API key, the environment, or both.

File: content/integrations/hosted/lightspeed_app/faq/disable-payment-methods.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title : "Can I disable payment methods?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+
+Yes, if you do not want certain payment methods to be displayed at all, you can disable them in the payment methods ordering list. You will see the disabled payment methods in the payment methods ordering list with a strikethrough.
+
+If you want different payment methods disabled for different languages, you can make these adjustments in the dropdown menu under the _Payment methods ordering_ header where you first need to select a country and then make change on a per language basis. _Default_ is used if no specific ruleset for a language is found.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/disable-payment-methods.md:9]: If you want different payment methods disabled for different languages, you can make these adjustments in the drop-down menu under the _Payment methods ordering_ header where you first need to select a country and then make changes on a per language basis. _Default_ is used if no specific rule-set for a language is found.

File: content/integrations/hosted/lightspeed_app/faq/disable-payment-methods.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+Are you missing any payment methods? Make sure that the payment methods are first enabled in the [MultiSafepay Control](https://docs.multisafepay.com/tools/multisafepay-control/) and then go to the settings page of our Lightspeed app and enable the respective payment method there again. By default newly activated payment methods in the MultiSafepay Control are disabled in the Lightspeed MultiSafepay Payments settings, so you need to enable them in both places.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/disable-payment-methods.md:11]: Are you missing any payment methods? Make sure that the payment methods are first enabled in the [MultiSafepay Control](https://docs.multisafepay.com/tools/multisafepay-control/) and then go to the settings page of our Lightspeed app and enable the respective payment method there again. By default, newly activated payment methods in the MultiSafepay Control are disabled in the Lightspeed MultiSafepay Payments settings, so you need to enable them in both environments.

File: content/integrations/hosted/lightspeed_app/faq/inline-buy-button.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+title : "Can I add a buy button near the payment methods?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+
+Yes, if you want to put an extra buy button near the selected payment method you can configure this in the settings page by going to _Inline Buy button_ and selecting _Show inline_ option.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/inline-buy-button.md:7]: Yes, if you want to put an extra buy button near the selected payment method, you can configure this in the settings page by going to _Inline Buy button_ and selecting _Show inline_ option.

File: content/integrations/hosted/lightspeed_app/faq/inline-news-letter.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title : "Can I show the newsletter signup in near the payment methods?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+
+Yes! If you want to put a newsletter checkmark near the selected payment method you can do so by going to _Inline newsletter_ on your settings page and choosing _Show inline_. The inline buy has to be enabled.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/inline-news-letter.md:7]: Yes. If you would like to put a newsletter check-mark near the selected payment method you can do so by going to _Inline newsletter_ on your settings page and choosing _Show inline_. The inline buy has to be enabled.

File: content/integrations/hosted/lightspeed_app/faq/maximal-order-amount.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+title : "Can I set a maximun order value for a payment method to show up?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+Yes, you can change the maximum order amount for a payment method to be shown in the checkout. To do so, you have to open the payment method in the _Payment methods ordering_ list by clicking on the "+" sign. Then you need to enter an amount in EUR cents into the maximum field. If you do not want a maximum amount enter “-1”.
+
+_For example, iDEAL with a maximum value of 1500 means that iDEAL will not be displayed in the checkout unless the total order amount is less then 15.00 EUR._
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/maximal-order-amount.md:8]: _For example, iDEAL with a maximum value of 1500 means that iDEAL will not be displayed in the checkout unless the total order amount is less then 15,00 EUR._

File: content/integrations/hosted/lightspeed_app/faq/minimal-order-amount.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+---
+title : "Can I set a minimal order value for a payment method to show up?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+
+Yes, you can change the minimal order amount for a payment method to be displayed to the customer. To do so, you have to open the payment method in the _Payment methods ordering_ list by clicking on the "+" sign and then entering an amount in EUR cents.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/minimal-order-amount.md:7]: Yes, you can change the minimum order amount for a payment method to be displayed to the customer. To do so, you have to open the payment method in the _Payment methods ordering_ list by clicking on the "+" sign and then entering an amount in EUR cents.

File: content/integrations/hosted/lightspeed_app/faq/minimal-order-amount.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+
+_For example, if you input a minimal value of 1500 for iDEAL, it means that iDEAL will not be displayed in the checkout unless the total order amount equals 15.00 EUR or more._
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/minimal-order-amount.md:9]: _For example, if you input a minimal value of 1500 for iDEAL, it means that iDEAL will not be displayed in the checkout unless the total order amount equals 15,00 EUR or more._

File: content/integrations/hosted/lightspeed_app/faq/minimal-order-amount.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+
+If you want different minimal order amounts for different languages you can configure this as well. In the dropdown menu under the _Payment methods ordering_ header you can select a country and make a change on a per language basis.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/minimal-order-amount.md:11]: If you want different minimum order amounts for different languages you can configure this as well. In the drop-down menu under the _Payment methods ordering_ header, you can select a country and make a change on a per language basis.

File: content/integrations/hosted/lightspeed_app/faq/order-payment-methods.md [hunk 11]
```diff
@@ -0,0 +1,10 @@
+---
+title : "Can I order the payment methods?"
+meta_title: "Lightspeed plugin refunding - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+
+Yes, The _Payment methods ordering_ section on the settings page allows you to change the order in which the payment methods are shown in the checkout. The order can be changed by dragging and dropping the items in the list. Do not forget to save the settings after making a change!
+
+If you want a different payment method order for different languages, you can configure it here as well. In the dropdown menu under the _Payment methods ordering_ header you can select a country and make a change on a per language basis.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/order-payment-methods.md:9]: If you want a different payment method order for different languages, you can configure it here as well. In the drop-down menu under the _Payment methods ordering_ header you can select a country and make a change on a per language basis.

File: content/integrations/hosted/lightspeed_app/faq/shop_id.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
+---
+title : "Where can I find my shop ID?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+1. Log into the /admin area of you webshopapp.
+2. Click in the bottom left corner on help.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/shop_id.md:7]: 2. Click in the bottom left corner on _Help_

File: content/integrations/hosted/lightspeed_app/faq/shop_id.md [hunk 13]
```diff
@@ -0,0 +1,8 @@
+---
+title : "Where can I find my shop ID?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+1. Log into the /admin area of you webshopapp.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/shop_id.md:6]: 1. Log into the /admin area of you webshop app

File: content/integrations/hosted/lightspeed_app/manual.md [hunk 14]
```diff
@@ -11,6 +11,13 @@ This is a beta version of the Lightspeed app, please contact the Integration tea
 
 {{% introduction_hosted "Lightspeed" %}}
 
+### Requirements
+The application is tested using the default, onestep and onepage checkout using the default thema.
+
+For the optimal experience of the user we included some Javascript and use our own images. This allows us to add, for example, a dropdown for iDEAL and our own icons for other payment methods. Some themas or users added scripts can cause issue, such as missing images for payment methods. To resolve these issue can require a developer.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/manual.md:17]: For the optimal experience of the user we included some Javascript and use our own images. This allows us to add, for example, a drop-down for iDEAL and our own icons for other payment methods. Some themes or users added scripts can cause issue, such as missing images for payment methods. To resolve these issue can require a developer.

File: content/integrations/hosted/lightspeed_app/manual.md [hunk 15]
```diff
@@ -11,6 +11,13 @@ This is a beta version of the Lightspeed app, please contact the Integration tea
 
 {{% introduction_hosted "Lightspeed" %}}
 
+### Requirements
+The application is tested using the default, onestep and onepage checkout using the default thema.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/manual.md:15]: The application is tested using the default, onestep and onepage checkout using the default theme.

File: content/integrations/hosted/lightspeed_app/manual.md [hunk 16]
```diff
@@ -11,6 +11,13 @@ This is a beta version of the Lightspeed app, please contact the Integration tea
...
+
+For the optimal experience of the user we included some Javascript and use our own images. This allows us to add, for example, a dropdown for iDEAL and our own icons for other payment methods. Some themas or users added scripts can cause issue, such as missing images for payment methods. To resolve these issue can require a developer.
+
+But don't worry if the javascript files are not loaded, all payment methods will keep working. And if you want to use our payment icons you can find them [here!](https://github.com/MultiSafepay/MultiSafepay-icons)
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/manual.md:19]: But don't worry if the javascript files are not loaded, all payment methods will keep working. If you want to use our payment icons you can find them [here!](https://github.com/MultiSafepay/MultiSafepay-icons)

**Review by Atiq-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/integrations/hosted/lightspeed_app/faq/disable-payment-methods.md (hunk 3 above)
Comment (Lars-MultiSafepay) [reply content/integrations/hosted/lightspeed_app/faq/disable-payment-methods.md]: I resolved all comments, can you take another look?

**Comment by Lars-MultiSafepay** [comment]
Not all banned words can be changed due to lightspeeds naming conventions.

File: content/integrations/hosted/lightspeed_app/faq/change-environment.md [hunk 17]
```diff
@@ -0,0 +1,8 @@
+---
+title : "Can I change the API key after installation?"
+meta_title: "Lightspeed plugin available payment methods - MultiSafepay Documentation Center"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+---
+
+Yes. On the settings page click on _Environment & API key_. You will be redirected to another page.
+Here you can update the API key, the environment, or both.
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/faq/change-environment.md:8]: On this page, the API key, environment, or both can be updated.

**Review by Atiq-MultiSafepay — CHANGES_REQUESTED** [review]

**Review by Qadira-MultiSafepay — APPROVED** [review]

File: content/integrations/hosted/lightspeed_app/manual.md [hunk 18]
```diff
@@ -11,6 +11,13 @@ This is a beta version of the Lightspeed app, please contact the Integration tea
 
 {{% introduction_hosted "Lightspeed" %}}
 
+### Requirements
+The application is tested using the default, onestep and onepage checkout using the default theme.
+
+For the optimal experience of the user we included some Javascript and use our own images. This allows us to add, for example, a drop-down for iDEAL and our own icons for other payment methods. Some themes or users added scripts can cause issue, such as missing images for payment methods. To resolve these issue can require a developer.
+
+But don't worry if the javascript files are not loaded, all payment methods will keep working. If you want to use our payment icons you can find them [here!](https://github.com/MultiSafepay/MultiSafepay-icons)
```
Comment (Atiq-MultiSafepay) [inline content/integrations/hosted/lightspeed_app/manual.md:19]: But don't worry if the javascript files are not loaded, all payment methods will continue to work. If you want to use our payment icons you can find them [here!](https://github.com/MultiSafepay/MultiSafepay-icons)

**Review by Atiq-MultiSafepay — CHANGES_REQUESTED** [review]

**Review by Atiq-MultiSafepay — APPROVED** [review]

**Pull request merged**